src/main/java/uk/ac/cam/cl/dtg/isaac/api/QuizFacade.java

```diff
@@ -1050,18 +1052,22 @@ public final Response getQuizAssignment(@Context final HttpServletRequest httpSe
 
             IsaacQuizDTO quiz = quizManager.findQuiz(assignment.getQuizId());
 
-            List<RegisteredUserDTO> groupMembers = this.groupManager.getUsersInGroup(group);
+            List<RegisteredUserDTO> groupMembers = this.groupManager.getUsersInGroup(group).stream()
+                    .sorted(Comparator.comparing(RegisteredUserDTO::getFamilyName))
+                    .sorted(Comparator.comparing(RegisteredUserDTO::getGivenName))
```

Comment:
```suggestion
                 .sorted(Comparator.comparing(RegisteredUserDTO::getFamilyName).thenComparing(RegisteredUserDTO::getGivenName))
```

This seems to be a suggested way of chaining, though your way is a bit nicer to read